app/.env.example

```diff
@@ -23,4 +24,5 @@ SALT=
 ALBERT_API_URL=
 ALBERT_USERNAME=
 ALBERT_API_TOKEN=
+USER_ALLOWED_DOMAINS='["ministere_example.gouv.fr", "ministere_example2.gouv.fr"]'
 COMPOSE_PROJECT_NAME=tchap-bot
```

Comment:
Why have duplicate environment names? Isn't it clearer to avoid duplicate names? We could have a section for additional environment variables used only by docker.

Reply:
Agreed, I followed what was done before. We should do that in a separate PR.